backend/app/services/knowledge/namespace_utils.py

```
@@ -0,0 +1,26 @@
+
+
+def is_organization_namespace(db: Session, namespace_name: str) -> bool:
+    """Return whether the namespace is an active organization namespace."""
+    if namespace_name == "default":
+        return False
+
+    namespace = (
+        db.query(Namespace.level)
+        .filter(
+            Namespace.name == namespace_name,
+            Namespace.is_active == True,
```

Ruff (0.15.7)

[error] Avoid equality comparisons to `True`; use `Namespace.is_active:` for truth checks

Replace with `Namespace.is_active`

(E712)